cves/kernel/CVE-2016-9754.yml

```diff
@@ -106,17 +108,17 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 7a8e76a3829f1067b70f715771ff88baf2fbf3c3
-  note: Discovered automatically by archeogit.
+  note: 'Discovered automatically by archeogit. Introduces the ring buffer.'
 - commit: 83f40318dab00e3298a1f6d0b12ac025e84e478d
```

Comment:
This bug was in the system for 4 years before being detected. I had a similar CVE with very old code containing a bug. Who knows how many undiscovered bugs are tucked away in large systems. Very interesting.